.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+(Thanks for sending a pull request! Please make sure you read the contribution
+guidelines [CONTRIBUTING.rst](CONTRIBUTING.rst), then fill out a PR like the
```

Comment:
This link gives you 404 page not found. I think you need to fix the reference to `CONTRIBUTING.rst` to the following:

`[CONTRIBUTING.rst](../CONTRIBUTING.rst)`

Or you can give a static HTTP reference to the CONTRIBUTING.rst file.